support/tarball/smoke-testNuGet.Config

```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
-    <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
-    <add key="source-built-packages" value="SOURCE_BUILT_PACKAGES" />
-    <add key="dotnet-core" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public%40Local/nuget/v3/index.json" />
     <add key="smoke-test feed" value="SMOKE_TEST_PACKAGE_FEED" />
+    <add key="source-built-packages" value="SOURCE_BUILT_PACKAGES" />
+    <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public%40Local/nuget/v3/index.json" />
```

Comment:
Should I just delete this file from source control? Should I place the generated version at some other location?

Reply:
I think it should be deleted.  What purpose would it server?  I think it is going to be a source of confusion?

Reply:
Thanks, that just reaffirms my preference to delete it.

Reply:
Okay. I have deleted it. The generated one now lives in `artifacts/tarball-smoke-testNuGet.Config`, and in the same location as before in the generated tarball.